crates/shared/src/rate_limiter.rs

```diff
@@ -174,7 +174,7 @@ impl RateLimiter {
     }
 }
 
-#[derive(Error, Debug, Clone, Default)]
+#[derive(Error, Debug, Clone, Default, PartialEq)]
```

Comment:
Is this required somewhere or just for completeness?

Reply:
Not required anymore for the latest code revision.